src/components/app/components/banner.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from "react";
+import { Link } from 'react-router-dom';
```

Comment:
I think there's no need to import `Link`,  there's one for the app already in `src/components/app/components/Applink`. For a better understanding you may checkout `Navbar.js` and `SideMenu.js`